src/handle_init_contract.c

```diff
@@ -36,7 +27,7 @@ void handle_init_contract(void *parameters) {
     memset(context, 0, sizeof(*context));
 
     uint32_t selector = U4BE(msg->selector, 0);
-    if (find_selector(selector, CIAN_SELECTORS, NUM_SELECTORS, &context->selectorIndex)) {
+    if (!find_selector(selector, CIAN_SELECTORS, NUM_SELECTORS, &context->selectorIndex)) {
         msg->result = ETH_PLUGIN_RESULT_UNAVAILABLE;
         return;
```

Comment:
I don't understand how this plugin could have worked before? (before your fix)

Reply:
Return values in their own `find_selector` function :
* found : 0
* not found : -1

In the `find_selector` function from the plugin SDK:
* found : true
* not found : false